library/src/main/kotlin/com/chuckerteam/chucker/internal/data/entity/HttpTransactionTuple.kt

```diff
@@ -17,6 +19,7 @@ internal data class HttpTransactionTuple(
     @ColumnInfo(name = "protocol") var protocol: String?,
     @ColumnInfo(name = "method") var method: String?,
     @ColumnInfo(name = "host") var host: String?,
+    @ColumnInfo(name = "requestHeaders") var requestHeaders: String?,
```

Comment:
Commenting here but valid for the whole PR:

1. I would not expose `requestHeaders` at all. It can be quite big and contains several unnecessary fields
2. I would instead create a new field called `graphQlOperationName` or `graphQlPath` and add it to both `HttpTransactionTuple` and `HttpTransaction`
3. You can then populate this inside the `RequestProcessor` so you'll just have a string that you can simply visualize. No need to inspect the whole Header set for what you need.

Also please add tests when you touch `RequestProcessor`

Reply:
I did an implementation first where I called `fun setGraphQlOperationName(headers: List<HttpHeader>)` method inside `fun setRequestHeaders(headers: List<HttpHeader>) {...}` but that felt like some hide knowledge and harder to test.

To make the methods with a single responsibility and clean structure I decided to split the calls and direct called on `RequestProcessor`

This created a repetition of the method `toHttpHeaderList(headers)` but I believe it will bring more safety for maintenance, let me know what you think 

(plus I didn't add any room version migration, but look ok)